src/save/properties.txt

```diff
@@ -1,28 +1,28 @@
 Mediterranean Avenue,Brown,60,50,2,10,30,90,160,250,30,0
 Baltic Avenue,Brown,60,50,4,20,60,180,320,450,30,0
-Reading Railroad,Railroad,200,0,25,50,100,200,0,100,0
+Reading Railroad,Railroad,200,0,25,50,100,200,0,100,0,0
 Oriental Avenue,Light Blue,100,50,6,30,90,270,400,550,50,0
 Vermont Avenue,Light Blue,100,50,6,30,90,270,400,550,50,0
 Connecticut Avenue,Light Blue,120,50,8,40,100,300,450,600,60,0
 St. Charles Place,Pink,140,100,10,50,150,450,625,750,70,0
 Electric Company,Utility,150,0,0,0,0,0,0,0,75,0
 States Avenue,Pink,140,100,10,50,150,450,625,750,70,0
 Virginia Avenue,Pink,160,100,12,60,180,500,700,900,80,0
-Pennsylvania Railroad,Railroad,200,0,25,50,100,200,0,100,0
+Pennsylvania Railroad,Railroad,200,0,25,50,100,200,0,100,0,0
 St. James Place,Orange,180,100,14,70,200,550,750,950,90,0
 Tennessee Avenue,Orange,180,100,14,70,200,550,750,950,90,0
 New York Avenue,Orange,200,100,16,80,220,600,800,1000,100,0
 Kentucky Avenue,Red,220,150,18,90,250,700,875,1050,110,0
 Indiana Avenue,Red,220,150,18,90,250,700,875,1050,110,0
 Illinois Avenue,Red,240,150,20,100,300,750,925,1100,120,0
-B&O Railroad,Railroad,200,0,25,50,100,200,0,100,0
+B&O Railroad,Railroad,200,0,25,50,100,200,0,100,0,0
```

Comment:
I will add another 0 between 200 and 100 at the end of the line to denote empty rentHotel value and remove the 0 from the end such that this parses correctly.